### PR TITLE
Pages 31-35

### PR DESCRIPTION
**Pull request by cameronshortt**
Pages 31-35 transcribed.  Page 33 had handwritten code, so `page033n-notes.txt` contains that code transcribed in the marked positions, with adjusted line numbers.

### CODE REVIEW

**Comment by cameronshortt** [comment]
For reference, lines 40, 59-65, and 79-82 in `page033n-notes.txt` were handwritten, and lines 75-76 had crossed out code, so are commented in `page033n-notes.txt`.  `page033n.txt` contains the exact transcriptions.

**Comment by cameronshortt** [comment]
Any unnumbered line in `page033n-notes.txt` was handwritten, and the crossed out code was left in like in the no-notes transcription.

**Comment by larsbrinkhoff** [comment]
Thanks, that was fast!  I'll have a look.

File: transcribed/page0310.txt [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+002     001 007         MOVE B,(C)
+003     001 052         TLO B,LSRMOD
+004     001 051         TLZ B,BADPC
+005     020 017         MOVEM B,UUOH
+006     118 046         MOVE B,TTYTBL(U)
+007     031 014         JUMPL B,ADISM1
+008     001 024         MOVEI C,TTYIF
+009                     SKIPN TINTC(B)
+010     031 014         JRST ADISM1
+011     118 033         TORM C,PIRQC(U)
```
Comment (larsbrinkhoff) [inline transcribed/page0310.txt:11]: IORM

File: transcribed/page0310.txt [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+039     001 016         POPJ P,
+040
+041     001 006 ASETMSK:        MOVE A,(J)      ;OPER 4
+042     001 025         TRZ A,BPIPT+VALRTF+BRKF+BCNTRZI ;MAKE SURE THESE GO TO SUP. PRO.
+043     118 036         MOVEM A,MSKST(U)
+044     001 050         MOVEI B,40+APRCHN
+045     001 027         TRNE A,AROIF
+046     001 050         MOVEI B,20+APRCHN
+047     118 025         MOVEM B,APRC(U)
+048     118 025         CONO APRC(U)
```
Comment (larsbrinkhoff) [inline transcribed/page0310.txt:48]: CONO @APRC(U)

File: transcribed/page0320.txt [hunk 3]
```diff
@@ -0,0 +1,55 @@
+001                                                                             PAGE 32
+002     118 041 ATRAND: CAML B,MEMTOP(U)
```
[the remark this answers is not in the document]
Comment (larsbrinkhoff) [reply transcribed/page0320.txt:2]: No!  Wrong line.

File: transcribed/page0331.txt [hunk 4]
```diff
@@ -0,0 +1,30 @@
+061     118 046         MOVEM D,TTYTBL(A)                                       PAGE 33.1
+062     118 046         LDB D,[220300,,TTYTBL(U)
+063     001 018         HRL D,U
+064     070 028         MOVSM D.TTYSTS(E)
```
Comment (larsbrinkhoff) [inline transcribed/page0331.txt:4]: Comma, not period.

File: transcribed/page0331.txt [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+078     118 008         MOVEM T,SIOCP(U)
+079
+080     001 016         POPJ P,
+081
+082     029 001 AIPOP:  PUSHJ P,ACLOSE  ;OPER 14
+083     118 008         MOVE T,SIOCP(U)
+084     001 017         TLNN T,-1
+085     021 049         JRST IOCERR
+086     001 017         POP T,(R)
+087     118 008         MOVEM T.SIOCP(U)
```
Comment (larsbrinkhoff) [inline transcribed/page0331.txt:27]: Comma, not period.

File: transcribed/page0340.txt [hunk 6]
```diff
@@ -0,0 +1,35 @@
+001     019 015 SUBTTL CORE ALLOCATOR                                           PAGE 34
+002
+003             UACORE:
+004     117 035 ACORE:  CAILE B,MXC     ;U HAS USER THAT WANTS CORE
+005     001 016         POPJ P,         ;IF U=USER,THIS IS IN CORE
+006     117 011         CAMN U,USER
+007     035 022         JUMPE B,ACRF
+008     001 006 ACRF1:  PUSH P,A
```
Comment (larsbrinkhoff) [inline transcribed/page0340.txt:8]: Missing comment ;MAY CLOBBER ALL ACS EX A,C,E,D,U

File: transcribed/page0350.txt [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+010     051 039         LDB E,[MUR,,MEMBLT(D)
+011     001 010         CAIN E,75       ;BLOCK ABOVE CURRENT FREE
+012     037 002         JRST ACORE6
+013     043 002 ACORE5: PUSHJ P,MXCCA   ;GET MAX CORE AVAIL IN BLOCK
+014     001 009         CAMG B,D        ;RETURN RESULT IN D
+015     044 003         JUMPN U,ACORE7  ;BLT GUY IN -- DONT BLT SYS IN HA HA
+016     118 041         SKIPN MEMTOP(U)
+017     044 003         JUMPE U,ACORE7  ;EXCEPT SOMETIMES HA HA
+018     002 002         MOVSI S,-MX1KB  ;INITITIATE MOBY SHUFFLE
+019     001 012         MOVEI I,O
```
Comment (larsbrinkhoff) [inline transcribed/page0350.txt:19]: Probably 0, not O.

File: transcribed/page0350.txt [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+020     037 019         JRST ACMS2
+021
+022     016 002 ACRF:   PUSHJ P,IODCL
+023     001 007         MOVEI B,O
```
Comment (larsbrinkhoff) [inline transcribed/page0350.txt:23]: 0

**Comment by larsbrinkhoff** [comment]
Merged, will add fixes in a separate commit.

**Pull request merged**